src/Xenial.Tasty/FeatureDetection/FeatureDetector.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using Xenial.Delicious.FeatureDetection.Win;
+
+namespace Xenial.Delicious.FeatureDetection
+{
+    internal static class FeatureDetector
+    {
+        internal static readonly string[] CmdProcessNames = new[]
```

Comment:
Perhaps `new HashSet<string>(StringComparer.CurrentCultureIgnoreCase) ` instead of array and use `Contains(name)` at line 32 instead? Might be _marginally_ faster and use more memory, but the intent is to keep the string comparison with the collection.

Reply:
Its a 2 or max 3 time call in app lifetime for now, i was considering a lazy, but its more a quality Feature for old windows cmd users, so i didn't bother too much